### ISSUE TITLE
Can you provide insights into the training process？

### ISSUE THREAD

**Issue by KANGX99** [zwx8981/LIQE#21]
Hi, thanks for your great work.
In the `train_unique_clip_weight.py` script(as shown in the image below), it appears that a single epoch doesn't utilize the entire training dataset. 
For example, when num_steps_per_epoch is set to 200, each batch consists of only 16 images from the Kadid-10k training set.
Therefore, within a single epoch, only 3200 images (200 steps * 16 images/step) are used for training, while the Kadid-10k training set contains 7125 images.
Is this a training trick? Because of the existence of shuffle in the parameter list of dalaloader.
![image](https://github.com/zwx8981/LIQE/assets/108923694/9ba29814-4d37-4f4e-a4ca-d4e57f020071)

**Comment by KANGX99** [comment]
OK, thank you for your explanation.
